packages/@aws-cdk/aws-dynamodb-global/lib/lambda-global-dynamodb.ts

```diff
@@ -0,0 +1,59 @@
+import cfn = require("@aws-cdk/aws-cloudformation");
+import iam = require("@aws-cdk/aws-iam");
+import lambda = require("@aws-cdk/aws-lambda");
+import cdk = require("@aws-cdk/cdk");
+import fs = require("fs");
+import path = require("path");
+import { DynamoDBGlobalStackProps } from "./aws-dynamodb-global";
+
+/**
+ * A stack that will make a Lambda that will launch a lambda to glue
+ * together all the DynamoDB tables into a global table
+ */
+export class LambdaGlobalDynamoDBMaker extends cdk.Stack {
+    /**
+     * The singleton Lambda function that will connect all the DynamoDB tables together into a global table
+     */
+    public lambdaFunction: lambda.SingletonFunction;
+
+    /**
+     * The content of the lambdaFunction (python3.7 using boto3)
+     */
+    public lambdaFunctionContent: string;
+
+    /**
+     * The CloudFormation CustomResource that will manage the lambda
+     */
+    public customResource: cfn.CustomResource;
+
+    constructor(scope: cdk.Construct, id: string, props: DynamoDBGlobalStackProps) {
+        super(scope, id, props);
+        const codeLocation = path.resolve(__dirname, "lambda", "handler.js.lambda");
+        this.lambdaFunctionContent = fs.readFileSync(codeLocation, "utf8");
+        this.lambdaFunction = new lambda.SingletonFunction(this, id + "-SingletonLambda", {
```

Comment:
Why append to the `id`, you could just use `SingletonLambda` as the `id`.

Reply:
I found it easier to track the deployed stacks in CFN by having a common prefix in the stack ID.  Otherwise, its difficult to keep track